modules/@apostrophecms/schema/index.js

```diff
@@ -790,6 +790,7 @@ module.exports = {
     self.addFieldType({
       name: 'object',
       async convert(req, field, data, destination) {
+        data = data[field.name];
```

Comment:
Not a big deal here, but avoiding to reassign is a good thing for code maintainability.